packages/web/src/components/user-list/UserList.tsx

```diff
@@ -59,18 +56,18 @@ const ConnectedUserList = (props: ConnectedUserListProps) => {
     props.onClickArtistName(handle)
   }
 
-  const { setPageSize, loadMore, pageSize } = props
+  const { loadMore, reset } = props
 
   useEffect(() => {
     // Load initially
-    setPageSize(pageSize)
     loadMore()
-  }, [setPageSize, loadMore, pageSize])
+  }, [loadMore])
 
-  // If page size changes, set it in the store
   useEffect(() => {
-    setPageSize(pageSize)
-  }, [setPageSize, pageSize])
+    return () => {
+      reset()
+    }
```

Comment:
```suggestion
    return reset
```

Although that may be less legible

Reply:
https://github.com/AudiusProject/audius-client/pull/1057#discussion_r825212701